File: Patika.dev Coderbyte Assessments (Weekly)/Week 4/Java/ThreeFiveMultiples.java
import java.util.*;
import java.io.*;

class Main {

  public static int ThreeFiveMultiples(int num) {
    // code goes here
    int sum = 0;
    for (int i = 3; i < num; i++) {
      if (i % 3 == 0 || i % 5 == 0){
        sum += i;
      }
    }
    return sum;
  }

  public static void main (String[] args) {
    // keep this function call here
    Scanner s = new Scanner(System.in);
    System.out.print(ThreeFiveMultiples(s.nextLine()));
  }

}

File: Patika.dev Coderbyte Assessments (Weekly)/Week 4/Python/Three Five Multiples.py
def ThreeFiveMultiples(num):

    multiples_list = []

    for number in range(1, num):
        if number % 5 == 0 or number % 3 == 0:
            multiples_list.append(number)

    # code goes here
    return sum(multiples_list)

# keep this function call here
print(ThreeFiveMultiples(input()))